website/docs/guides/migration/versions/02-upgrading-to-v1.5.md

```diff
@@ -63,3 +63,10 @@ More to come!
 
 ### dbt-core Python API
 - Auto-generated documentation ([#2674](https://github.com/dbt-labs/docs.getdbt.com/issues/2674)) for dbt-core CLI & Python API for programmatic invocations
+
+
+### Materialized views and tests
+
+:::caution Under construction 🚧
```

Comment:
@dataders actually is this suppose to have a link to anything? (eg. https://github.com/dbt-labs/dbt-core/discussions/7213)?

Reply:
Hey @dataders wanted to check in with this?